extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -115,6 +128,58 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
       (event) => event.stopPropagation(),
       true
     );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (currPageId) => {
+        const isPostState = this.rootEl_.classList.contains(
+          'i-amphtml-story-interactive-post-selection'
+        );
+        if (isPostState) {
+          // If it's already been interacted with, do not animate.
+          return;
+        }
+        if (currPageId != this.getPageEl().getAttribute('id')) {
+          // Resets animation when navigating away.
+          cancelAnimationFrame(this.currentRAF_);
+          clearTimeout(this.landingAnimationDelayTimeout_);
+          this.inputEl_.value = 0;
+          this.onDrag_();
+          this.rootEl_.classList.remove(
+            'i-amphtml-story-interactive-mid-selection'
+          );
+          return;
+        }
+        let startTime;
+        const animateFrame = (currTime) => {
+          // Set current startTime if not defined.
+          if (!startTime) {
+            startTime = currTime;
+          }
+          const elapsed = currTime - startTime;
+          if (HINT_ANIMATION_DURATION_MS < elapsed) {
+            this.rootEl_.classList.remove(
+              'i-amphtml-story-interactive-mid-selection'
```

Comment:
Can you declare a top level constant in `...abstract.js` called `MID_SELECTION_CLASS` and another `POST_SELECTION_CLASS` for this value (and the post-selection classname) so that we don't have to redefine it everywhere? Then you can import it in this file and reuse them (would be great if you also find occurrences of these variables and replace them in the other files as well)